portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,24 @@
 <!DOCTYPE html>
 <html>
+  <link href="https://fonts.googleapis.com/css2?family=Cookie&display=swap" rel="stylesheet">
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link rel="stylesheet" href="photo-frame.css">
+    <link rel="stylesheet" href="buttons.css">
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <div id="greeting-container">
```

Comment:
I think the indentation changed here. Did you use a different editor?

Reply:
Done.